### PR TITLE
Generate controllers

### PR DESCRIPTION
**Pull request by ishimwezachee**
In this milestone I have done the following;

-  Created a new app. Give it a meaningful name.
-  Initialized your project with Git.
- Added controllers
-  Added a Postgres database set up.

-  For each URL you should create:
    - A route.
    - An action in the correct controller.
    - A view file.

-  View files should include only placeholders written in plain HTML, e.g:
    - Here is a list of posts for a given user

-   added only 4 required URLs.

### CODE REVIEW

**Review by julie-ify — CHANGES_REQUESTED** [review]
Hi @ishimwezachee, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

- [x] You have submitted a project, that has no descriptive README. Kindly add a descriptive README using Microverse [template](https://github.com/microverseinc/readme-template/blob/master/README.md).

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: README.md [hunk 1]
```diff
@@ -1,24 +1,73 @@
...
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- A special thanks to Microverse for inspiring this project. 
+
+## 📝 License
+
+This project is [MIT](./LICENCE) licensed.
```
Comment (MahmoudBakr23) [inline README.md:73]: - Please fix the MIT link for a completely professional repo

**Review by MahmoudBakr23 — APPROVED** [review]
Hi @ishimwezachee 😃, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! 🎉 

- Well done setting up your application database, controllers, and routes 🚀 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (ishimwezachee) [reply README.md:73]: Got it, thank you for the reviewer

**Pull request merged**